README.md

```diff
@@ -88,7 +88,7 @@ server and the VSCode extension from sources.
    [node.js](https://nodejs.org/), and install `yarn` via:
 
    ```bash
-   npm install yarn
+   npm install -g yarn
    yarn install
    ```
 
```

Comment:
Could you add a little comment line right below that says ("Note this will install `yarn` globally.") ?

(Same in `building-superbol.rst`)